chain/client/src/client_actor.rs

```diff
@@ -218,6 +227,21 @@ pub fn start_client(
     )
     .unwrap();
     let resharding_handle = client.chain.resharding_handle.clone();
+
+    let maybe_gc_arbiter = {
```

Comment:
I don't think it's this straight forward to create a new actor. We need to handle creating mocks for tests as well.